controllers/admin/AdminAjaxPsxMktgWithGoogleController.php

```diff
@@ -462,6 +474,86 @@ public function getShopAttributes()
         );
     }
 
+    /*******************
+     * Product filters *
+     *******************/
+
+    public function getProductFilterOptions(array $inputs)
+    {
+        if (!isset($inputs['kind'])) {
+            http_response_code(400);
+            $this->ajaxDie(json_encode([
+                'success' => false,
+                'message' => 'Missing kind key',
+            ]));
```

Comment:
I reckon nothing else should be done after this, right ?
so why not having a return ?